spec/factories/application_letters.rb

```diff
@@ -8,20 +8,27 @@
 #  event_id    :integer          not null
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
+#  status      :integer          not null
 #
 
 FactoryGirl.define do
   factory :application_letter do
     motivation "MyString"
     user
     event
-    status nil
   end
```

Comment:
Maybe rather :pending for the status then nil? since pending is the default state after an application letter was created.